src/imex/key_transfer.rs

```diff
@@ -141,7 +141,7 @@ fn create_setup_code(_context: &Context) -> String {
 
     for i in 0..9 {
         loop {
-            random_val = rng.gen();
+            random_val = rng.r#gen();
```

Comment:
This is a deprecated alias for `random()`, maybe use it instead?
https://rust-random.github.io/rand/src/rand/rng.rs.html